source/code-generation/query-converter/convert-queries.txt

```diff
@@ -27,6 +27,7 @@ About this Task
 Steps
 -----
 
+
```

Comment:
[question] is this extra line necessary?

Reply:
@ltran-mdb2, Unneeded. I sometimes add blank lines to start creating the PR for build checks. Fixed.